src/clips-specs/rcll-central/goal-production.clp

```diff
@@ -541,7 +528,7 @@
   (bind ?goal
     (goal-tree-assert-central-run-all-prio PRODUCE-ORDER ?*PRODUCTION-C0-PRIORITY*
       (goal-production-assert-deliver ?wp-for-order ?order-id ?instruct-parent ?ds)
-      (goal-production-assert-mount-cap ?wp-for-order ?cs ?bs INPUT ?order-id)
+      (goal-production-assert-mount-cap ?wp-for-order ?cs ?col-cap ?order-id)
```

Comment:
maybe we should change it to cap-col for consistency with other parts of the code?

Reply:
fixed it throughout the code